corehq/apps/reports/static/reports/js/edit_submission.js

```diff
@@ -2,6 +2,7 @@ hqDefine("reports/js/edit_submission", function () {
     $(function () {
         var initialPageData = hqImport("hqwebapp/js/initial_page_data").get;
         window.MAPBOX_ACCESS_TOKEN = initialPageData('mapbox_access_token'); // maps api is a global variable depended on by touchforms
+        window.DEFAULT_GEOCODER_LOCATION = initialPageData('default_geocoder_location');
```

Comment:
I don't think you need this, and also think you can remove the line above since you got rid of the `window`-related logic.